[docs] - Minor formatting & copy cleanup in multiple agents doc (#19147)
dagster-io/dagster: docs/content/dagster-cloud/deployment/agents/running-multiple-agents.mdx

  agentLabel: "My agent" # optional, only supported on 0.13.14 and later
```

</TabItem>
<TabItem name="Amazon ECS">

### In Amazon ECS

The `isolated_agents` option can be set as per-deployment configuration on the `dagster.yaml` file used by your agent. See the [ECS configuration reference](/dagster-cloud/deployment/agents/amazon-ecs/configuration-reference#per-deployment-configuration) guide for more information.

</TabItem>
</TabGroup>

---

## Routing requests to specific agents

Every Dagster Cloud agent serves requests from one or more queues. By default, requests for each code location are placed on a default queue and your agent will read requests only from that default queue.

> In some cases, you might want to route requests for certain code locations to specific agents. For example, you might want to route requests for one code location to an agent running in an on-premise data center, but requests for all other code locations to an agent running in AWS. 

Reviewer: Nit that doesn't need to be addressed since I know you're focused on cleaning up: these back to back sentences start with `you want to`, which tripped up my head. Leaving a suggestion, but take or leave. tis all kosher!

```suggestion
There are reasons to route requests for certain code locations to specific agents. For example, you might want to route requests for one code location to an agent running in an on-premise data center, but requests for all other code locations to an agent running in AWS. 
```

> To route requests for a code locations to a specific agent, annotate the code locations with the name of a custom queue and configure an agent to serve only requests for that queue.

Reviewer: ```suggestion
To route requests for a code location to a specific agent, annotate the code location with the name of a custom queue and configure an agent to serve only requests for that queue.
```
